cloudify/dispatch.py

```diff
@@ -673,8 +673,7 @@ def _execute_workflow_function(self):
             self.ctx.internal.start_local_tasks_processing()
             result = self.func(*self.args, **self.kwargs)
             if not self.ctx.internal.graph_mode:
-                tasks = list(self.ctx.internal.task_graph.tasks_iter())
-                for workflow_task in tasks:
+                for workflow_task in self.ctx.internal.task_graph.tasks:
```

Comment:
Why was this tasks_iter(), and why doesn't it need to be any more?

Reply:
this looked at every task, for reasons I describe in the `task .get: remove from graph` commit; it still does, I'm just changing the method